src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("Hello, world!");
```

Comment:
nice that says hello world! We could probably do without the comma.

Reply:
Alright will remove thanks!